### ISSUE TITLE
Space Types library item collapses after dragging in Space Type from library

### ISSUE THREAD

**Issue by macumber** [NREL/OpenStudio#2767]
Have to keep opening this to drag in more space types

**Comment by ljbrackney** [comment]
Agree.  I believe Construction Sets have the same issue.  Irritating.

**Comment by ljbrackney** [comment]
This needs to be escalated.  Now that we're adding larger (DEER) libraries that are needed in real workflows, this becomes a significant PITA.
